cdk/dcm_floodlight/transform.js

```diff
@@ -57,7 +58,32 @@ function trackPostMapper(input, mappedPayload, rudderContext) {
 
   const baseEndpoint = "https://ad.doubleclick.net/ddm/activity/";
 
-  let event = getValueFromMessage(message, "event");
+  let event;
+  if (message.type === "page") {
+    const { category } = message.properties;
```

Comment:
Post mapper is missing js docs plus this can be made more understandable with inline docs

Reply:
added inline comment